plugins/netskope/plugin.spec.yaml

```diff
@@ -0,0 +1,413 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: netskope
+title: Netskope
+description: This plugin allows users to create, read, update, and delete URL list
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+supported_versions: ["2022-02-11"]
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/netskope
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.netskope.com/
+tags:
+  - netskope
+  - cloud
+  - security
+hub_tags:
+  use_cases: [cloud_security]
+  keywords: [endpoint]
+  features: []
+enable_cache: false
+connection:
+  tenant:
+    title: Tenant
+    description: Owner's name that is included in URL
+    type: string
+    required: true
+    example: exampletenant
+  api_key_v1:
+    title: API Key v1
+    description: Netskope Authorization Token
+    type: credential_secret_key
+    required: true
+    example: j5740ps1cbukyk3t8kib3wa36aq2v3da
```

Comment:
These are not real keys, right? Please replace both API v1 and v2 example keys with the example secret key credential from the [Style Guide](https://docs.rapid7.com/insightconnect/style-guide#input-examples).

Reply:
Yes. There are not real ones. Updated.